.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea/
 _vendor/
+vendor/
```

Comment:
Please add new line at end and add Gopkg.lock file here and remove file Gopkg.lock